app/services/explorer.js

```diff
@@ -893,39 +894,38 @@ export default Ember.Service.extend({
      * @param {DS.Store} store
      * @return {Ember.RSVP.Promise} result of the AJAX call
      */
-    getKeyList(bucket, store) {
-        var clusterId = bucket.get('clusterId');
-        var bucketTypeId = bucket.get('bucketTypeId');
-        var bucketId = bucket.get('bucketId');
-        var explorer = this;
+    getKeyList(bucket, store, start=1, rows=100) {
```

Comment:
Alter the default value in `rows` if you want to test this out locally and want the pagination chunks to be shorter. This value gets set as an attribute on the model, and the component picks it up from there.